tests/src/test/java/org/teavm/classlib/java/util/EnumSetTest.java

```diff
@@ -196,6 +196,214 @@ public void retainAll() {
         assertEquals(original, set);
     }
 
+    @Test
+    public void iterator() {
```

Comment:
This test does not pass in WebAssembly in optimized mode. This is likely to some bug, not related to this particular commit, so please just suppress WebAssembly and WASI targets for this test